docs/networking/dns.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+title: Domain Name System
+nav_exclude: false
+has_children: false
+parent: Networking
+has_toc: true
+search_exclude: false
+last_modified_date: 2023-02-03
+---
+# Understanding DNS and How to Revert to Standard DNS Providers
+
+{: .no_toc}
+{% include toc.md %}
+
+## What is DNS?
+
+[DNS](/docs/learning/terms#domain-name-system), or the Domain Name System, is the system that translates human-readable domain names (e.g. www.google.com) into machine-readable IP addresses (e.g. 216.58.194.174). It acts as a sort of phonebook for the internet, allowing users to access websites and other online resources using easy-to-remember names instead of difficult-to-remember numerical addresses.
+
+## List of DNS Providers
+
+*You can use any of the following depending on your Personal Preference.*
+
+* **Google Public DNS:** 8.8.8.8 and 8.8.4.4
+* **Cloudflare:** 1.1.1.1 and 1.0.0.1
+* **OpenDNS:** 208.67.222.222 and 208.67.220.220
+* **Quad9:** 9.9.9.9 and 149.112.112.112
+
+## How to Revert to Standard DNS Providers
+
+The process of reverting to a standard DNS provider varies depending on the operating system you are using. Here are the steps for some common operating systems:
+
+### Windows
+
+1. Click the Start button and type "Control Panel" in the search bar.
+2. Click "Network and Sharing Center."
+3. Click "Change adapter settings."
+4. Right-click the network connection you want to change and select "Properties."
+5. Select "Internet Protocol Version 4 (TCP/IPv4)" and click "Properties."
+6.  Select "Use the following DNS server addresses" and enter the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers).
+7. Click OK to save the changes.
+
+### MacOS
+
+1. Click the Apple menu and select "System Preferences."
+2. Click "Network."
+3. Select the network connection you want to change and click "Advanced."
+4. Click the "DNS" tab.
+5. Click the "+" button to add a new DNS server address and enter the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers).
+6. Click OK to save the changes.
+
+### Linux
+
+The specific steps to change the DNS server in Linux depend on the distribution you are using. For most distributions, you can change the DNS server by editing the `/etc/resolv.conf` file and adding the preferred DNS server addresses for the provider [you want to use](#list-of-dns-providers).
```

Comment:
Regarding [what was asked in the development channel](https://discord.com/channels/749314018837135390/1071069467955703888/1071760874135629835), it's a hard answer to ask on what's the most terse way to explain how to change the DNS that covers all the distros. We probably want people to stay within the GUI, so we could mention the few places where to find the settings in a couple common DEs (KDE, Gnome, Mate are a few that come to mind) and as a last resort suggest finding what [network manager](https://wiki.archlinux.org/index.php?title=Network_configuration&oldid=766537#Network_managers) their system is using then use that to modify DNS.

Reply:
Though for desktop geared distros, it's almost always going to be `NetworkManager` that handles networking.